frontend/add-client/add-client.component.tsx

```diff
@@ -274,12 +274,13 @@ ${mediaDesktop} {
   max-width: 35rem;
 }
 
-& label {
+& label, & [role=group] {
```

Comment:
checkboxes have to be grouped together, and one way is with `role="group"`

Reply:
I already did -- see https://github.com/JustUtahCoders/comunidades-unidas-internal/pull/244/files#diff-1ac7f78a5eb8e5786d6803da39b116d5R46